GVFS/GVFS.UnitTests.Windows/Windows/Upgrader/UpgradeOrchestratorWithIProductUpgraderTests.cs

```diff
@@ -0,0 +1,202 @@
+﻿using GVFS.Common;
+using GVFS.Common.Git;
+using GVFS.Tests.Should;
+using GVFS.UnitTests.Mock.Common;
+using GVFS.UnitTests.Windows.Mock.Upgrader;
+using GVFS.Upgrader;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+
+namespace GVFS.UnitTests.Upgrader
+{
+    [TestFixture]
+    public class UpgradeOrchestratorWithIProductUpgraderTests
```

Comment:
Can this just be renamed to something like `UpgradeOrchestratorTests`?   Is there any functionality in `UpgradeOrchestrator` is not being covered by these tests?

(It's safe to assume we're mocking out the components used by `UpgradeOrchestrator`, there's no need to call out `IProductUpgrader` specifically)